STYLE_GUIDE.md

```diff
@@ -274,6 +294,7 @@ Composer uses [Google Style Docstrings](https://sphinxcontrib-napoleon.readthedo
 
 The following guidelines apply to documentation.
 1.  Each function that needs a docstring must have its input arguments and return statement (if not None) annotated.
```

Comment:
And `Raises`?

Reply:
Added a comment that all custom exceptions should be documented.